skfda/ml/classification/_qda.py

```diff
@@ -207,7 +207,10 @@ def _fit_gaussian_process(
 
             cov_estimators.append(cov_estimator)
             means.append(cov_estimator.location_)
-            covariance.append(cov_estimator.covariance_.data_matrix[0, ..., 0])
+            # TODO: QDA should use the covariance estimators interface
```

Comment:
Currently, QDA assumes that 'covariance_' is an 'FDataGrid'.
However, following this pull request, implementing RKHS and the Mahalanobis distance, will allow us to work only using the interface provided by the 'CovarianceEstimator' class.

This is simply a temporal fix until QDA is refactored, when adding the Mahalanobis distance.